django_freeradius/api/serializers.py

```diff
@@ -90,7 +90,16 @@ class RadiusBatchSerializer(serializers.ModelSerializer):
     pdf = serializers.FileField(required=False, read_only=True)
     number_of_users = serializers.IntegerField(required=False,
                                                write_only=True,
-                                               min_value=1)
+                                               min_value=1,)
```

Comment:
why did you add this comma? Can you remove it please?

Reply:
My bad, @nemesisdesign . I'll remove it just now.